### PR TITLE
Ddcnos-6102: Deceased page update

### PR DESCRIPTION
**Pull request by lewey**
- [x] Unit tests passing
- [x] Coverage reached
- [x] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/deceased_helpline.scala.html [hunk 1]
```diff
@@ -14,70 +14,43 @@
...
 ){
 
     <header class="page-header">
-        <h1 id="deceased-header" class="flush--bottom">@Messages("tai.deceased.heading", user.getDisplayName)</h1>
+        <h1 id="deceased-header" class="flush--bottom">@Messages("tai.deceased.heading.title")</h1>
     </header>
 
-    <p id="deceased-p1">@Messages("tai.deceased.information.p1", user.getDisplayName)</p>
+    <div class="grid-row">
+        <div class="grid grid-2-3">
```
Comment (thirlz95) [inline app/views/deceased_helpline.scala.html]: Just a note, looked into main_template which is using article. I believe it's possible to inject these classes; grid grid-2-3 classes to save overruling with using divs

File: app/views/deceased_helpline.scala.html [hunk 2]
```diff
@@ -14,70 +14,33 @@
...
-@()(implicit request: Request[_], user: controllers.auth.AuthedUser, messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
-
-@import uk.gov.hmrc.urls.Link
+@()(implicit request: Request[_], messages: Messages, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, partialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
 
 @main(
-    title = Messages("tai.deceased.title"),
-    articleClasses = Some("full-width"),
-    authedUser = Some(user),
+    title = Messages("tai.deceased.heading.title"),
```
Comment (danewatts) [inline app/views/deceased_helpline.scala.html]: Use implicit messages

File: conf/messages.cy [hunk 3]
```diff
@@ -320,36 +320,19 @@ tai.currentYearSummary.heading=Gwirio''ch Treth Incwm
...
-tai.deceased.tell.us=Dweud Wrthym Unwaith
-tai.deceased.textphone.number=0300 200 1900
-tai.deceased.textphone=Ffôn testun:
-tai.deceased.title=Llinell gymorth trethi yn dilyn profedigaeth
+tai.deceased.telephone.advice=Dwedwch ‘profedigaeth’ pan ofynnir i chi pam yr ydych yn galw.
+tai.deceased.textphone.title=Ffôn testun:
+tai.deceased.textphone.number=0300 200 3319
+tai.deceased.outsideUK.title=Y tu allan i’r DU:
+tai.deceased.outsideUK.number=+44 300 200 1900
+tai.deceased.openingTimes.mondayToFriday=Dydd Llun i ddydd Gwener: 08:30 – 17:00
```
Comment (danewatts) [inline conf/messages.cy]: As discussed, can we get a screenshot of this message please

File: app/views/deceased_helpline.scala.html [hunk 4]
```diff
@@ -22,7 +24,7 @@
 ){
 
     <header class="page-header">
-        <h1 class="flush--bottom">@messages("tai.deceased.heading.title")</h1>
+        <h1 class="flush--bottom">@Html(s"${messages("tai.deceased.heading.title.part1")} ${HtmlFormatter.htmlNonBroken(messages("tai.deceased.heading.title.part2"))}") </h1>
```
Comment (Tateni93) [inline app/views/deceased_helpline.scala.html]: Does this need to be wrapped by HTML

Comment (lewey) [reply app/views/deceased_helpline.scala.html]: I believe so

**Pull request merged**